keps/sig-api-machinery/2876-crd-validation-expression-language/README.md

```diff
@@ -228,6 +225,22 @@ kind: CustomResourceDefinition
               type: integer
 ```
 
+Example Validation Rules:
```

Comment:
We have many lists that are logically keyed by a name field.  While not adviseable, items in some these lists are special.  Can you provide an example of checking the value a pod envvar named "foo"?

Reply:
Sure, added:

`!('MY_ENV' in self.envars) || self['MY_ENV'].matches('^[a-zA-Z]*$')`